Sources/Extensions/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -289,3 +289,15 @@ public extension Array where Element: Equatable {
 	}
 
 }
+
+// MARK: - Methods (CustomStringConvertible)
+public extension Array where Element: CustomStringConvertible {
+    
+    /// SwifterSwift: Returns a String joined by new line.
+    ///
+    ///     ["James", "How are you?"].paragraph() -> "James\nHow are you?")
+    ///
+    public func paragraph() -> String {
+        return self.map{ $0.description }.joined(separator: "\n")
```

Comment:
You can do this using the `joined(separator:)` operator. 
```swift
 ["James", "How are you?"].joined(separator: "\n") // -> "James\nHow are you?"
```
So, I don't see any value using this extension. 😕